---
layout: default
title: schaefer lab / research
categories:
 - research
---

<div class="row">
	<div class="col-md-12">
		<div class="smallhead">


	     <h2>Modes of selection</h2>
       <img src="/images/research/darwin_finches.jpg" width=200 align=left hspace="40"></img>
	     <p>We are interested how alterations of the DNA transform a healthy into a malignant cell. 10,000s of human genomes and epigenomes
          have been sequenced over the last years. These efforts uncovered a large number of genetic and epigenetic alterations in cancer patients.
          Given this large number of alterations it is difficult to tell which of those contribute to disease progression. In particular, we are
          interested to understand how to detect selection for alterations that are not point mutations such as epigenetic changes or structural
          alterations of the genome or negative selection that preserves the function of a gene instead of changing it.</p>
        </div>
    	</div>
    </div>

    <div class="row">
    	<div class="col-md-12">
    		<div class="smallhead">
<br>See for example:</ br>
<br><a href="/papers/zapata-negative-selection/" class="off">Zapata et al. 2018. Genome Biology.</a></ br>
<br><a href="/papers/dmitrijeva-methylation/" class="off">Dmitrijeva et al. 2018. NAR.</a></ br>

		</div>
	</div>
</div>


<div class="row">
	<div class="col-md-12">
		<div class="smallhead">



      <h2>Cancer: a complex system</h2>
      <img src="/images/logo.png" width=200 align=left hspace="40"></img>
      <p>We aim to understand how the environment impacts selection by comparing the selective pressure in cancers under different environmental
        conditions. Why are certain genes only found mutated in one type of tissue but not in others? How does selection change from tissue to tissue,
        after treatment, in metastasis and as a consequence of microbial infection? </p>
        <p>We start to understand that a tumor consists of many different parts but that the whole tumor is more than the sum of its parts. We aim to understand the
          composition of a tumor beyond heterogeneity of malignant cells (for example, how do immune cells, microbes and fibroblasts integrate into this
          ecosystem) and how do system level properties of the cancer (such as resistance against therapy or migratory behaviour) emerge through interactions
          of the parts of the tumor.</p>
        </div>
    	</div>
    </div>

    <div class="row">
      <div class="col-md-12">
        <div class="smallhead">
          <br>See for example:</ br>
          <br><a href="/papers/schaefer-tissue/" class="off">Schaefer and Serrano. 2016. Scientific Reports.</a></ br>
          <br><a href="/papers/cramer-CNA/" class="off">Cramer et al. 2016. eLife.</a></ br>



		</div>
	</div>
</div>

<div class="row">
	<div class="col-md-12">
		<div class="smallhead">

      <h2>Tools for the age of precision medicine</h2></p>
      <img src="/images/research/tools.jpg" width=200 align=left hspace="40"></img>
      <p>Traditionally, genes have been studied in isolation. However, genes and proteins act together to create complex phenotypes. We develop resources and methods
        using tools from network and systems biology to better understand the interplay between genes. One particular interest of us is to better the understanding of how
        the rewiring of cellular signaling pathways leads to diseases such as cancer. We do so by developing bias reduced networks with high physiological relevance and
        appyling subnetwork finding algorithms to multi-omics cancer networks. </p>

      </div>
    </div>
</div>

<div class="row">
	<div class="col-md-12">
		<div class="smallhead">

      <br>See for example:</ br>
      <br><a href="/papers/schaefer-bias/" class="off">Schaefer et al. 2015. Frontiers in Genetics.</a></ br>
      <br><a href="/papers/alanis-hippie/" class="off">Alanis-Lobato et al. 2016. NAR.</a></ br>
		</div>
	</div>
</div>
